release-notes/releases-index.json

```diff
@@ -12,7 +12,8 @@
             "support-phase": "preview",
             "eol-date": "2026-05-11",
             "release-type" : "sts",
-            "releases.json": "https://dotnetcli.blob.core.windows.net/dotnet/release-metadata/9.0/releases.json"
+            "releases.json": "https://dotnetcli.blob.core.windows.net/dotnet/release-metadata/9.0/releases.json",
+            "supported-os.json": "https://raw.githubusercontent.com/dotnet/core/main/release-notes/8.0/supported-os.json"
```

Comment:
Richard and I discussed - we'll also sync the supported-os.json to the same storage location as releases.json.

Reply:
Moved to https://github.com/dotnet/core/pull/9367